cmd/common/monitor_test.go

```diff
@@ -1,50 +0,0 @@
-package common
```

Comment:
Don't we require this anymore?

Reply:
Monitor file is already removed. So I don't think, this is required anymore.